.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+  pull_request: ~
+  workflow_dispatch: ~
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js v16
+        uses: actions/setup-node@v3
+        with:
+          node-version: "16.x"
+      - name: Install dependencies
+        run: npm i
+      - name: Build tokens
+        run: npm run build:tokens
+      - name: Build icons
+        run: npm run build:icons
+      - name: Formatting
+        run: npm run format
```

Comment:
`npm run format` actually formats the files and writes them to disk. in CI we want to actually just run a check on the format. so we want another script like `format:check` that runs `prettier . --check` instead of `prettier . --write`

Reply:
derp, missed that hah